benchmarks/json/bench_json.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shell script to bench json parsers over different documents
+
+source ../bench_common.sh
+source ../bench_plot.sh
+
+## CONFIGURATION OPTIONS ##
+
+# Default number of times a command must be run with bench_command
+# Can be overrided with 'the option -n'
+count=5
+
+## HANDLE OPTIONS ##
+
+function init_repo()
+{
+	echo "Preparing submodules"
+	git submodule update --init
+	echo "Submodules ready"
+	mkdir -p inputs
+	echo "Preparing data for benchmarks"
+	if [ ! -e inputs/large_escaped.json ]; then
+		echo "Downloading file 1/4"
+		wget -O inputs/large_escaped.json https://github.com/seductiveapps/largeJSON/blob/master/100mb.json?raw=true
+	fi
+	echo "File 1/4 ready"
+	if [ ! -e inputs/magic.json ]; then
+		echo "Downloading file 2/4"
+		wget -O inputs/magic.json http://mtgjson.com/json/AllSets-x.json
+	fi
+	echo "File 2/4 ready"
+	if [ ! -e inputs/twitter.json ]; then
+		echo "Downloading file 3/4"
+		wget -O inputs/twitter.json https://github.com/miloyip/nativejson-benchmark/raw/master/data/twitter.json
+	fi
+	echo "File 3/4 ready"
+	if [ ! -e inputs/gov_data.json ]; then
+		echo "Downloading file 4/4"
+		wget -O inputs/gov_data.json https://edg.epa.gov/data.json
```

Comment:
auto-concatenate the files gov_data.json and twitter.json in order to have biger files.

Eg something like
~~~bash
echo "[" > big_gov_data.json
for i in $(seq 10); do
  test "$i" != "1" && echo "," >> big_gov_data.json
  cat gov_data.json >> big_gov_data.json
done
echo "]" >> big_gov_data.json
~~~